releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -6,6 +6,33 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
 
+## 1.2.0 - January 13, 2020
+
+* Simplified Feature Management config file format.
+
+```yaml
+feature-management:
+  Beta: true
+  DarkTheme:
+    EnabledFor:
+      -
+        Name: Random
+        Parameters:
+          chance: "50"
+  BetaAB:
+    EnabledFor:
+      -
+        Name: Random
+        Parameters:
+          chance: "50"
+      -
+        Name: ClientFilter
+        Parameters:
+          clientIp: 10.0.0.1
+```
+
+* The isEnabled method is now non-blocking Async and has been renamed to isEnabledAsync using Reactor.
```

Comment:
Please quote APIs, for example, `isEnabled`.

Reply:
Do you mean the API for the value we return which is Mono? i.e. https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html?

Reply:
I mean use single backticks so \`isEnabled\` will look like `isEnabled`.